mmpose/datasets/datasets/body3d/body3d_semi_supervision_dataset.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from torch.utils.data import Dataset
+
+from mmpose.datasets.builder import DATASETS, build_dataset
+
+
+@DATASETS.register_module()
+class Body3DSemiSupervisionDataset(Dataset):
+    """Mix Dataset for semi-supervised training in 3D human pose estimation
+    task.
+
+    The dataset combines data from two datasets and
```

Comment:
We can give a more specific description, e.g. the two datasets include a labeled one and an unlabeled one.